examples/testing-ground/src/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { ScatterChart } from '@iot-app-kit/react-components';
+import { LineChart, WebglContext } from '@iot-app-kit/react-components';
+import { initialize } from '@iot-app-kit/source-iotsitewise';
+import { render } from 'react-dom';
+import { environment } from './utils/environment';
+
+const { query } = initialize({
+  awsCredentials: {
+    accessKeyId: environment.AWS_ACCESS_KEY_ID,
+    secretAccessKey: environment.AWS_SECRET_ACCESS_KEY,
+    sessionToken: environment.AWS_SESSION_TOKEN,
+  },
+  awsRegion: 'us-west-2',
+  settings: {
```

Comment:
I think we should be able to omit this (if not, I'll add an issue to make it so we can)